python/kwiver/vital/algo/resection_camera.cxx

```diff
@@ -0,0 +1,50 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include <pybind11/pybind11.h>
+#include <python/kwiver/vital/algo/trampoline/resection_camera_trampoline.txx>
+#include <python/kwiver/vital/algo/resection_camera.h>
+
+namespace kwiver {
+namespace vital  {
+namespace python {
+namespace py = pybind11;
+
+void resection_camera(py::module &m)
+{
+  py::class_< kwiver::vital::algo::resection_camera,
+              std::shared_ptr<kwiver::vital::algo::resection_camera>,
+              kwiver::vital::algorithm_def<kwiver::vital::algo::resection_camera>,
+              resection_camera_trampoline<> >( m, "ResectionCamera" )
+    .def(py::init())
+    .def_static("static_type_name",
+                &kwiver::vital::algo::resection_camera::static_type_name)
+    .def("resection",
```

Comment:
In order to properly declare an overload for use in Python [this](https://pybind11.readthedocs.io/en/stable/classes.html#overloaded-methods) syntax must be used. The convention so far seems to utilizing static casting so one approach may look like: `.def("resection",  static_cast< kwiver::vital::camera_perspective_sptr (kwiver::vital::algo::resection_camera::*)
           (std::vector<kwiver::vital::vector_2d> const&,
            std::vector<kwiver::vital::vector_3d> const&,
            std::vector<bool>&,
            kwiver::vital::camera_intrinsics_sptr) const)>(&kwiver::vital::algo::resection_camera::resection));` or rather than a static cast, some built in pybind11 sytax can be used.

Reply:
I've addressed this.  I initially copied the syntax from other vital algorithms which were also using the C-style cast.  I agree that `static_cast` is better.  I've updated that here, but not in any other algorithms.